blinds/blinds_schedule.py

```diff
@@ -330,6 +337,49 @@ def hasConflict( timeBlockList ):
 
         return False
 
+    '''
+    Parses a timezone formatted like GMT-0600 and returns a pytz timezone for it. 
+    Currently does not support half hour differences from GMT, as these are not included in the 
+    list of supported timezones in pytz for Etc/GMT-X. While they do exist for the specific place names, the conversion 
+    is much more difficult.
+    '''
+    @staticmethod
+    def tzFromGmtString( gmtStr ):
+        tzRegex = r"GMT([+-])(\d{4})" # matches things like GMT-0600
+        gmtTimezoneBase = "Etc/GMT"
+        
+        matches = re.search( tzRegex, gmtStr)
+
+        if not matches:
+            raise InvalidTimeZoneStringException( "Invalid timezone string: " + gmtStr )
+
+        plusminus = matches.group(1)
+        offset = int( matches.group(2) ) / 100
```

Comment:
In the conditional below you check if offset < -14 but you only ever have the offset being positive. You have to use plusminus when creating offset.

Reply:
Fixed.